dockerfiles/pbench/controller/run.sh

```diff
@@ -63,8 +63,19 @@ elif [[ $JOB == "http" ]]; then
 	git clone https://github.com/jmencak/http-ci-tests.git /root/http-ci-tests
 	cd /root/http-ci-tests
 	source /opt/pbench-agent/profile; . ./http-test.sh all
-elif [[ $JOB == "mongo" ]]; then
-	source /opt/pbench-agent/profile; 
+elif [[ $JOB == "scaleup" ]]; then
+        echo "Running ocp scaleup"
+        git clone https://github.com/chaitanyaenr/scale-ci-ansible.git /root/scale-ci-ansible
```

Comment:
In practice we should be tying this to the redhat-performance namespace rather than individual user namespace.  Is there a specific requirement at this time to clone from local fork rather that the original repo?  To extend this, I would like to get away from the practice of git cloning in code as it leads to non-deterministic/greater difficulty in reproducing any tests.  I understand from a realistic sense right now we can not necessarily get away from that practice as it is tech debt at this point.

Reply:
scale-ci-ansible has all the stuff we need for scaleiup, I didn't notice that it was using my namespace. Updated the PR. Thanks.

Reply:
Agree that we need to avoid using the forks wherever possible.

Reply:
Got it, Yup at least for now we can avoid forks, and work on reducing (if possible the git cloning inside of tests in the future.)